examples/issue-30.rs

```diff
@@ -60,20 +64,24 @@ fn main() {
         Ok(_) => {}
     };
     // Parse the XML into a RNode
-    // content[0] is the root node
-    let root = Document::try_from((srcxml, None, None))
-        .expect("unable to parse XML")
-        .content[0]
-        .clone();
-    let mut doc = NodeBuilder::new(NodeType::Document).build();
-    doc.push(root).expect("unable to append root node");
+    let root = Rc::new(SmiteNode::new());
+    xmlparse(root.clone(), srcxml.as_str(), None, None).expect("unable to parse XML");
```

Comment:
```suggestion
    xmlparse(root.clone(), srcxml.as_str(), None).expect("unable to parse XML");
```

following the change in commit [5077833bdddd2b1a4780893fc498d4808a70806b](https://github.com/ballsteve/xrust/commit/5077833bdddd2b1a4780893fc498d4808a70806b#diff-0eabaaeb1e42c9e6de94c303f7fd31aeba580aa78bc0c0e66f6ac97506e47b46L31-R31)